### PR TITLE
Adds wedding/costume/bloody armor crates and other tweaks

### PR DESCRIPTION
**Pull request by Henri215**
## What Does This PR Do

Makes a non super hero version of E-N suit.
Changed costume crate name to "Service Costume crate".
Added Costume crate (30 points), Wedding crate (50 points), Bloody armor crate (contraband, security crate, 30 points).
Tweaks Justice crate by adding 1 helmet justice escape variation.
Contents at images of changes.
Suggestions of items to be added or removed are welcome.
## Why It's Good For The Game

Its good because gives usage to items that only existed in code and that could be used as new crates contents.
A benefit of adding the bloody armor crate is to give more options for the cargo console contraband setting (wich currently only has 2 crates).

## Images of changes

Costume crate
costumecraa

Wedding crate
weddingcrate

Bloody Armor crate
armoor

Justice crate
justice crate

## Testing

- Ordered the crates and verified the contents.
## Changelog
:cl:
add: Added costume and wedding crates
add: Added bloody armor crate as contraband
add: Added justice helmet variation to standard justice enforcer crate
tweak: Old costume crate renamed to "service costume crate"
/:cl:

### CODE REVIEW

**Comment by ElorgRHG** [comment]
**Soul**

but anyways, the wedding, costume stuff and adding that random justice helmet rename are all fine,
the bloody vest and lord admiral stuff are... 

They're... not great and should be slated for removal rather than put front and center like this. They also do not have species variants so will look likewise pretty bad on less humanoid species, mainly vox. 

And adding just a, opinion time, worse looking vest, sprite quality wise and onmob security wise, crate that's 80 points that's ALSO in the contraband section and will come in a locked box...  that's an interesting decision
<details>

image

image

</details>

then again, aspriring spriters yells at cloud so whatever.

File: code/datums/supplypacks.dm [hunk 1]
```diff
@@ -254,6 +255,15 @@ GLOBAL_LIST_INIT(all_supply_groups, list(SUPPLY_EMERGENCY,SUPPLY_SECURITY,SUPPLY
...
 	containername = "armor crate"
 
+/datum/supply_packs/security/armor/bonus
+	name = "Bloody Armor Crate"
+	contains = list(/obj/item/clothing/suit/armor/vest/bloody,
+					/obj/item/clothing/suit/armor/vest/bloody,
+					/obj/item/clothing/suit/armor/vest/bloody)
+	cost = 30
+	containername = "bloody armor crate"
+	contraband = 1
```
Comment (Sirryan2002) [inline code/datums/supplypacks.dm:265]:
```suggestion
	contraband = TRUE
```

**Review by Sirryan2002 — COMMENTED** [review]
I worry this will contribute to supply pack bloat, at some point we need to start breaking up the security and misc sections, they both have like 10-20 crates

**Review by Sirryan2002 — APPROVED** [review]
code is fine

File: code/modules/clothing/head/misc.dm [hunk 2]
```diff
@@ -327,6 +327,10 @@
 	item_state = "chickensuit"
 	flags = BLOCKHAIR
 
+/obj/item/clothing/head/corgi/en
+	name = "E-N suit head"
+	icon_state = "enhead"
+
```
Comment (SteelSlayer) [inline code/modules/clothing/head/misc.dm:333]: This seems to be already defined under a different path:

image

File: code/modules/clothing/suits/miscellaneous.dm [hunk 3]
```diff
@@ -172,6 +172,10 @@
 	flags_inv = HIDESHOES|HIDEJUMPSUIT
 	dog_fashion = /datum/dog_fashion/back
 
+/obj/item/clothing/suit/corgisuit/en
+	name = "\improper super-hero E-N suit"
+	icon_state = "ensuit"
+
```
Comment (SteelSlayer) [inline code/modules/clothing/suits/miscellaneous.dm:178]: Same here:

image

**Review by SteelSlayer — CHANGES_REQUESTED** [review]

**Review by SteelSlayer — APPROVED** [review]

**Pull request merged**